### PR TITLE
DUI: Review and rationalize XAML files

### PR DESCRIPTION
**Pull request by aosyatnik**
This PR attempts to review XAML, move common styles to resources, so that they can be reused; move colors to resources, so that they can be stored at the single place.

Reviewers
@Benglin , please take a look.

Link to YouTrack:
[MAGN-5334](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-5334) DUI: Review and rationalize XAML files

### CODE REVIEW

File: src/DynamoCoreWpf/Controls/StandardPanel.xaml [hunk 1]
```diff
@@ -5,106 +5,114 @@
...
-                    Value="True" />
-            <Setter Property="FocusVisualStyle"
-                    Value="{x:Null}" />
-            <Setter Property="Visibility"
-                    Value="{Binding Path=Visibility, Converter={StaticResource BoolToVisibilityCollapsedConverter}}" />
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.SidebarGridDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
```
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/StandardPanel.xaml]: I think you can remove `SidebarGridDictionaryUri`, and probably the other two also (need to try and see if it still works).

File: src/DynamoCoreWpf/Controls/StandardPanel.xaml [hunk 2]
```diff
@@ -5,106 +5,114 @@
...
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
+        </ResourceDictionary>
     </UserControl.Resources>
     <UserControl.Visibility>
         <Binding Path="ClassDetailsVisibility"
-                 Converter="{StaticResource BoolToVisibilityCollapsedConverter}" />
+                 Converter="{StaticResource BooleanToVisibilityCollapsedConverter}" />
```
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/StandardPanel.xaml]: Is `BooleanToVisibilityCollapsedConverter` defined? I only saw `BoolToVisibilityCollapsedConverter` in the latest  code.

**Comment by Benglin** [comment]
Hi @aosyatnik, thanks a lot for taking the trouble to rationalize these XAML files, I have tried the changes and it is all fine! LGTM :)

File: src/DynamoCoreWpf/Controls/StandardPanel.xaml (hunk 2 above)
Comment (aosyatnik) [inline src/DynamoCoreWpf/Controls/StandardPanel.xaml]: Now it's in `DynamoConvertersDictionaryUri`.

**Pull request merged**